src/Stomp/Network/Observer/Heartbeat/Emitter.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/*
+ * This file is part of the Stomp package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Stomp\Network\Observer\Heartbeat;
```

Comment:
Yes, when I added it I thought about the `Monitor` but there will be never more than two, so it's fine to Prefix them.